pack/return/rttcu_encounter.json

```diff
@@ -20,5 +539,569 @@
+        "encounter_code": "return_to_in_the_clutches_of_chaos",
+        "encounter_position": 7,
+        "faction_code": "mythos",
+        "illustrator": "Cristi Balanescu",
+        "name": "Merchant District",
+        "pack_code": "rttcu",
+        "position": 55,
+        "quantity": 1,
+        "shroud": 4,
+        "text": "[action]: Move 1 breach from Merchant District to the current act. You may spend any number of charges, secrets, supplies, or ammo from any assets you control to move that many additional breaches.",
+        "traits": "Arkham.",
+        "type_code": "location"
+    },
+    {
+        "back_text": "- When building the act deck, replace the original version of act 3 with the new version included in this encounter set. Remove the original version from the game.\n- When building the Cosmos, include the 3 new Cosmos location included in the <i>Return to Before the Black Throne</i> encounter set.\nWhen the game begins, the investigators may remove 3 resources from the scenario reference card to have each investigator begin the game with a Nightgaunt Steed story asset in play. Remove each remaining Nightgaunt steed from the game.",
+        "code": "54056",
+        "double_sided": true,
+        "encounter_code": "return_to_before_the_black_throne",
+        "encounter_position": 1,
+        "faction_code": "mythos",
+        "name": "Return to Before the Black Throne",
+        "pack_code": "rttcu",
+        "position": 56,
+        "quantity": 1,
+        "text": "Perform the setup as indicated in the <i>The Circle Undone</i> Campaign Guide, with the following exceptions:\n- When gathering encounter sets, also gather the new encounter sets for <i>Return to Before the Black Throne</i> shown here.\n<b>(Continued on reverse side.)</b>",
+        "type_code": "scenario"
+    },
+    {
+        "back_flavor": "You stand on a precipice before the Black Throne and gaze up at the blind, voiceless monstrosity that is Azathoth. The mere sight of it causes your vision to blur and your body to shake uncontrollably. The monotonous piping is so loud now, it rattles your bones and causes blood to drip fro your eyes and ears. Should Azathoth awaken, it will devour every world that ever was.",
+        "back_name": "The End",
+        "back_text": "Check the Campaign Log.\n- If at least two of the following are true, you may proceed to <b>(→R2)</b>:<i>You accepted your fate</i>, <i>Mesmerizing Flute</i> is listed under\"Mementos Discovered\", and <i>Ritual Components</i> is listed under \"Mementos Discovered\"\n-If a least two if the following are true, you map proceed to <b>(→R3)</b>: <i>You rejected your fate</i>, <i>Scrap of Torn Shadow</i> is listed under \"Mementos Discovered,\" and <i>Wisp of Spectral Mist</i> is listed under \"Mementos Discovered\"\n- If <i>Strange Incantation</i> and <i>Bloody Tree Carvings</i> are listed under \"Mementos Discovered\", you pay proceed to <b>(→R6)</b>\n- Otherwise, <b>(→R4)</b>",
```

Comment:
add space between 'under' and Mementos:

`under \"Mementos...` [for Mesmerizing Flute]